fabfile.py

```diff
@@ -90,3 +90,9 @@
     run_command,
     update_packages,
 )
+from fabric.api import env
+
+hosts = env['hosts']
+with open("ansible_runner/inventory/hosts", "w") as file:
+    for host in hosts:
+        file.write(host + '\n')
```

Comment:
do we need a method to remove the added hosts from this file?

Reply:
Probably, at the moment it overwrites the file each time but clearing it each time is probably better